### DISCUSSION TITLE
Menu Changed To Tune By Itself

### DISCUSSION THREAD

**Discussion by Cal-LF** [Jyers/Marlin#283] [Q&A] [answered]
I am using E3V2-ManualMesh-3x3-v4.2.2.bin and while printing, the menu will change to Tune by itself. This has happened while watching. It stays on the Tune menu highlighted on Back. It does not return to the main menu by itself. Clicking Back will return to the main menu.

**Comment by vhristenco** [comment]
This also happens to me, randomly, on every print. Clicking "back" will return to the main screen, but the printer enters the "tune" menu again, after a while.

**Comment by Jyers** [comment]
I really wish I had more data on this issue. It seems pretty few and far between and I can't for the life of me imagine what could cause it.

**Reply by Cal-LF** [reply]
Still happens here. Checked all wire connections to the display. More of an odd occurrence than anything else. Maybe it is a feature so adjustments can be made quicker. :)

**Comment by nullstalgia** [comment, marked as answer]
Yeah if I had to guess it's probably EMF from the motors/motor drivers clicking the encoder knob. This isn't firmware specific, fortunately (or unfortunately, depending on how you look at it).

I think I had it on Pause once and it was saved from pausing by the confirm dialog box :P

**Reply by gothcha** [reply]
Hi I had this problem with Creality firmware, it beeped and went to tune and back constantly during prints. I realised it was the press switch on the encoder being very sensitive. I turned the printer off and pressed the knob firmly loads of times and this cleared the fault! I think there may be debris inside the mechanism randomly  triggering the button.

**Comment by Cal-LF** [comment]
Referring to nullstalgia answer. Agree this is not firmware related.
The printer is not self clicking now that it is raining in our area. Static must be building from the motion of the rubber belts. If the steppers are not bonded to the frame, a charge could be triggering the screen encoder. Another possible issue is the poor or no grounding of the power plug allowing a random spike to cause a trigger. Such as a furnace or fridge or other appliance to spike the power supply.
